quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/cli/command/UpdateCompatibilitySave.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2025 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.keycloak.quarkus.runtime.cli.command;
+
+import java.io.File;
+import java.io.IOException;
+
+import org.keycloak.quarkus.runtime.cli.PropertyException;
+import org.keycloak.quarkus.runtime.compatibility.ServerInfo;
+import org.keycloak.util.JsonSerialization;
+import picocli.CommandLine;
+
+@CommandLine.Command(
+        name = UpdateCompatibilitySave.NAME,
+        description = "Stores the metadata necessary to determine if a configuration is compatible."
+)
+public class UpdateCompatibilitySave extends AbstractUpdatesCommand {
```

Comment:
Could we make this so that it's something like `UpdateCompatibilityMetadata`, which by default just prints the info to stdout and then if a flag is provided we optionally write to a file? I think this would make things easier to debug locally.